fix mt7615e: probe of 0000:02:00.0 failed with error -89 in chaos_calmer
--- a/mt7615/pci.c
+++ b/mt7615/pci.c
@@ -33,10 +33,6 @@
 
 	pci_set_master(pdev);
 
-	ret = pci_alloc_irq_vectors(pdev, 1, 1, PCI_IRQ_ALL_TYPES);
-	if (ret < 0)
-		return ret;
-
 	ret = pci_set_dma_mask(pdev, DMA_BIT_MASK(32));
 	if (ret)
 		return ret;
@@ -52,7 +48,6 @@
 	struct mt7615_dev *dev = container_of(mdev, struct mt7615_dev, mt76);
 
 	mt7615_unregister_device(dev);
-	pci_free_irq_vectors(pdev);
 }
 
 struct pci_driver mt7615_pci_driver = {
